src/index.tsx

```diff
@@ -180,6 +184,27 @@ const VisGraph = forwardRef<
     };
   }, [events, network]);
 
+  useEffect(() => {
+    if (!zoomKey || !network) {
+      return () => {};
```

Comment:
why not just `return`?

Reply:
There's an eslint rule that was on the previous project that's not on here: consistent return type, which usually requires ugly disable comments or just returning a no-op function